provision/vvv-init.sh

```diff
@@ -32,22 +32,33 @@ if [[ ! -d "${VVV_PATH_TO_SITE}/public_html" ]]; then
   cd ${VVV_PATH_TO_SITE}/public_html/src/
   echo "Creating wp-config.php for src.wordpress-develop.dev and build.wordpress-develop.dev."
   noroot wp core config --dbname=wordpress_develop --dbuser=wp --dbpass=wp --quiet --extra-php <<PHP
-// Match any requests made via xip.io.
-if ( isset( \$_SERVER['HTTP_HOST'] ) && preg_match('/^(src|build)(.wordpress-develop.)\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(.xip.io)\z/', \$_SERVER['HTTP_HOST'] ) ) {
+// Match any requests made via xip.io or tests.wordpress-develop.dev.
+if ( isset( \$_SERVER['HTTP_HOST'] ) && preg_match('/^tests\.wordpress-develop\.dev$|^(src|build)(\.wordpress-develop\.)\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(.xip.io)\z/', \$_SERVER['HTTP_HOST'] ) ) {
```

Comment:
I suppose this could be better as:

```
'/^(tests\.wordpress-develop\.dev|^(src|build)(\.wordpress-develop\.)\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(.xip.io))\z/'
```